.gitignore

```diff
@@ -6,6 +6,8 @@ config/config.secret.exs
 /doc
 .DS_Store
 
+#elixir LS files
+/.elixir_ls
 # IntelliJ IDEA files
```

Comment:
Sorry I missed this on the first passthrough, but more formatting issues here

- there should be a space in the comment: `#elixir LS files` -> `# elixir LS files`
- there should be a blank line between lines 10 and 11